cime_config/allactive/config_pesall.xml

```diff
@@ -2453,4 +2714,78 @@
       </pes>
     </mach>
   </grid>
+  <grid name="a%T62.+oi%oARRM60to10|a%TL319.+oi%oARRM60to10">
```

Comment:
@apcraig for example I'm not sure how these two layouts are new here, in particular the one for ARRM60to6 below. Maybe you had removed these at some point and I took the wrong change during conflict resolution?

Reply:
Not sure either.  I haven't removed them.  I will create a sandbox and test as well.  May take me a day or two to get to it.